tests/seatest_tests.c

```diff
@@ -72,5 +72,5 @@ void all_tests(void)
 
 int main(int argc, char** argv)
 {
-  return run_tests(all_tests) ? 0 : 1;
+  return run_tests(all_tests);
```

Comment:
An eagle eyed friend of mine has pointed out that this change could backfire, since an exit status is stored in a single byte. I'll push a fixed version soon.